### ISSUE TITLE
Connection leak after fs2 update

### ISSUE THREAD

**Issue by reactivemaster** [evotor/aecor-postgres-journal#13]
**Environment**
```
JDK version: 1.8.0_152-b16
Scala version: 2.12.7
Database: Postgres 11.2
Driver version: postgresql-42.2.5
Aecor version: 0.18.0
Aecor Postgres Journal version: 0.3.0
```

I tried to build my eventsourced application based on Aecor and everything was fine when I used fs2 version 1.0.0.
When I updated fs2 to version 1.0.4, then I got connection leak in Hikari pool.

In my logs I began to see the following:
```
15:28:25.276 DEBUG c.z.h.pool.HikariPool HikariPool-1 - Pool stats (total=25, active=25, idle=0, waiting=32)
15:28:47.905 DEBUG c.z.h.pool.HikariPool HikariPool-1 - Timeout failure stats (total=25, active=25, idle=0, waiting=31)
15:28:47.907 DEBUG c.z.h.pool.HikariPool HikariPool-1 - Timeout failure stats (total=25, active=25, idle=0, waiting=31)
15:28:47.912 DEBUG c.z.h.pool.HikariPool HikariPool-1 - Timeout failure stats (total=25, active=25, idle=0, waiting=31)
15:28:47.912 DEBUG c.z.h.pool.HikariPool HikariPool-1 - Timeout failure stats (total=25, active=25, idle=0, waiting=30)
```

I investigated a bit and I came to the fact that the problem disappears if I replace `.stream` to `.to[List]` in the `PostgresEventJournal` methods (of course, with a slight change of further transformation logic).
Not sure, maybe I just don't know some specific settings that can fix it all.

**Comment by vpavkin** [comment]
@reactivemaster 
We faced this one too, luckily it just got localized and fixed here:
https://github.com/tpolecat/doobie/pull/879

Hopefully it's released soon, and then we'll release a fixed journal

**Comment by notxcain** [comment]
Fixed in #16
